mmv1/products/filestore/examples/ansible/instance.yaml

```diff
@@ -24,6 +24,7 @@ task: !ruby/object:Provider::Ansible::Task
       - network: default
         modes:
           - MODE_IPV4
+        connect_mode: DIRECT_PEERING
```

Comment:
I changed this one from connectMode to connect_mode as well @rileykarson 

Not sure if that's needed for Ansible

Reply:
Per https://github.com/GoogleCloudPlatform/magic-modules/pull/5161#discussion_r702096826 no change needed!